pkg/backend/httpstate/backend.go

```diff
@@ -1243,22 +1302,11 @@ func (b *cloudBackend) renderAndSummarizeOutput(
 	renderer := display.NewCaptureProgressEvents(
 		stack.Ref().Name(),
 		op.Proj.Name,
-		display.Options{
-			ShowResourceChanges: true,
-		},
+		display.Options{},
```

Comment:
Why has this changed?

Reply:
It can be big and in the error-summary case we truncate quite a lot to reduce size and make it fast etc, so most of it will be lost for any update that changes a few resources.

However I'll look into it more and do it another PR if I think it helps 👍